SPECS/php/php.spec

```
@@ -914,7 +823,7 @@ cp %{SOURCE50} %{SOURCE51} %{SOURCE53} .
 
 %build
 # Set build date from https://reproducible-builds.org/specs/source-date-epoch/
-export SOURCE_DATE_EPOCH=$(date +%s -r NEWS)
+export SOURCE_DATE_EPOCH=$(date +%{s} -r NEWS)
```

This is probably the linter being a bit too eager to find RPM macros where there are none.:)

```suggestion
export SOURCE_DATE_EPOCH=$(date +%s -r NEWS)
```